File: src/server/gkill/api/req_res/add_mi_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type AddMiResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	AddedMi *reps.Mi `json:"added_mi"`

	AddedKyou *reps.Kyou `json:"added_kyou"`
}

File: src/client/classes/api/req_res/add-mi-response.ts
'use strict'

import { Mi } from '@/classes/datas/mi'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class AddMiResponse extends GkillAPIResponse {

    added_mi: Mi

    added_kyou: Kyou | null

    constructor() {
        super()
        this.added_mi = new Mi()
        this.added_kyou = null
    }

}


